test/integration/webhook/v1alpha2/clusterqueue_test.go

```diff
@@ -87,14 +89,29 @@ var _ = ginkgo.Describe("ClusterQueue Webhook", func() {
 			var updateCQ kueue.ClusterQueue
 			gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(cq), &updateCQ)).Should(gomega.Succeed())
 			updateCQ.Spec.Resources = []kueue.Resource{
-				*testing.MakeResource("@cpu").Obj(),
+				*testing.MakeResource("@cpu").Flavor(testing.MakeFlavor("x86", "5").Obj()).Obj(),
 			}
 
 			err := k8sClient.Update(ctx, &updateCQ)
 			gomega.Expect(err).Should(gomega.HaveOccurred())
 			gomega.Expect(errors.IsForbidden(err)).Should(gomega.BeTrue(), "error: %v", err)
 		})
 
+		ginkgo.It("Should have valid resources quantity when creating", func() {
```

Comment:
consider using a single `ginkgo.DescribeTable` for all the validation errors to reduce code.

where necessary, you can use an anonymous functions to do the loops.